### PR TITLE
Provide the way to specify 2FA expiration policy. KC-220

### PR DESCRIPTION
**Pull request by sk-keeper**

### CODE REVIEW

File: keepercommander/loginv3.py [hunk 1]
```diff
@@ -506,13 +511,39 @@ def handleTwoFactor(params: KeeperParams, encryptedLoginToken, login_resp):
...
+            while not otp_code:
+                if show_duration:
+                    show_duration = False
+                    prompt_exp = '\n2FA Code Duration: {0}.\nTo change duration: 2fa_duration=login|30_days|forever' \
+                        .format('Require Every Login' if mfa_expiration == proto.TWO_FA_EXP_IMMEDIATELY else
+                                'Save on this Device Forever' if mfa_expiration == proto.TWO_FA_EXP_NEVER else
+                                'Ask Every 30 days')
+                    print(prompt_exp)
+
+                answer = input('\n' + prompt_str + ': ')
```
Comment (dstromberg) [inline keepercommander/loginv3.py:526]: Why does the user have to enter 2fa_duration=whatever?  Are we planning for more options here someday?

Comment (sk-keeper) [reply keepercommander/loginv3.py:526]: This "2fa_duration=whatever" is optional. User does not have to enter it. The default value will be assumed if  2fa_duration is omitted. Yes, this area may be extended with additional options.

**Review by maksimu — COMMENTED** [review]
Was it documented?

**Review by dstromberg — APPROVED** [review]
LGTM

**Review by maksimu — APPROVED** [review]
Was it documented?

**Pull request merged**